src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/Microsoft.VisualStudio.ProjectSystem.Managed.VS.csproj

```diff
@@ -1,6 +1,6 @@
 ﻿<!-- Copyright (c)  Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information. -->
 <Project Sdk="Microsoft.NET.Sdk">
-  <Import Project="..\VisualStudio.props"/>
+  <Import Project="..\VisualStudio.props" />
```

Comment:
Can we remove unnecessary changes like this?

Reply:
Hmm. Looks like we dirtied the project when I added the file - and that reformatted the doc. Given the current design of MSBuild (it goes through the XML API) - this isn't possible to avoid this change. Is that your question or was it "back out this change"?

Reply:
I am fine with this as its going to happen anyway the next time someone edits this doc.  Just curious why it was edited at all as there were no changes made to the content.

Reply:
Due to the way globbing works; CPS "dirties" the project to trigger a reevaluation. We have a bug on this.